.github/workflows/test.yaml

```diff
@@ -14,11 +14,7 @@ jobs:
         go-version: [1.22.x]
         os: [ubuntu-latest, macos-latest, windows-latest]
         include:
```

Comment:
`include` can be deleted, since `ubuntu-latest` was covered by line 15

Reply:
Fixed 👍